regression/cbmc/pointer-primitive-check-01/test.desc

```diff
@@ -1,15 +1,43 @@
-CORE
+CORE broken-smt-backend
```

Comment:
Why `broken-smt-backend` when it wasn't before?

Reply:
@martin-cs I am not sure _why_ but some individual pointer checks seem to have different results in the smt backend. Because in the previous version of this we AND-ed them all together this didn’t matter before.

Reply:
@hannes-steffenhagen-diffblue Thanks for the explanation but this does leave me a little concerned.  They really shouldn't do.  This is kind of important functionality.